### PR TITLE
Fixing Windows paths

### PR DESCRIPTION
**Pull request by m0mus**
Signed-off-by: Dmitry Kornilov <dmitry.kornilov@oracle.com>

### CODE REVIEW

**Comment by m0mus** [comment]
Fixes https://github.com/oracle/helidon-build-tools/issues/79

File: sitegen-maven-plugin/src/main/java/io/helidon/build/sitegen/maven/AbstractAsciiDocMojo.java [hunk 1]
```diff
@@ -264,8 +265,12 @@ void processFile(
      * @return PathMatchers for the globs
      */
     private static Stream<PathMatcher> pathMatchers(Path inputDirectory, String[] globs) {
-          return Arrays.stream(globs)
+        return Arrays.stream(globs)
                 .map(glob -> {
+                    if (WINDOWS) {
+                        String pattern = "glob:" + inputDirectory + File.separator + glob.replace("/", "\\");
+                        return FileSystems.getDefault().getPathMatcher(pattern.replace("\\", "\\\\"));
```
Comment (danielkec) [inline sitegen-maven-plugin/src/main/java/io/helidon/build/sitegen/maven/AbstractAsciiDocMojo.java]: `Matcher.quoteReplacement(String.valueOf(inputDirectory));` may also work

Comment (m0mus) [reply sitegen-maven-plugin/src/main/java/io/helidon/build/sitegen/maven/AbstractAsciiDocMojo.java]: Sure. Thanks for pointing on it. I am not sure that using it brings any advantages over `String.replace`.

**Comment by romain-grecourt** [comment]
Thank you for fixing this !

**Pull request merged**